### PR TITLE
adding wire message to the notification

### LINKED ISSUE #1554: Notification for new Wire message are missing the wire message...
As a user receiving instant notification, I need the email message to be complete and include the wire message in the notification. 
As it is now, the notification text is missing the wire message and I have to click on the link to reply to the wire in order to see the wire message. 

In English and French text are missing words (missing text in bold): 
EN: Sebastien.Lemay posted a new wire **message** entitled 
FR: Sebastien.Lemay a ajouté un nouveau **message sur le fil** intitulé 

Please add wire message in notification and fix french language. 
wire notifications

### PR DESCRIPTION
**Pull request by pandurx**
fixes gctools-outilsgc/gcconnex#1554

now displays image in the email notification

### CODE REVIEW

File: mod/cp_notifications/views/default/cp_notifications/email_template.php [hunk 1]
```diff
@@ -358,6 +358,13 @@
 		$cp_topic_description_discussion_en = gc_explode_translation($cp_topic_description_discussion,'en');
 		$cp_topic_description_discussion_fr = gc_explode_translation($cp_topic_description_discussion,'fr');
 
+
+		if ($vars['cp_topic']->getSubtype() === 'thewire') {
+			$cp_notify_msg_title_en = elgg_echo('cp_notify:body_new_content:title3', array($vars['cp_topic']->getOwnerEntity()->getURL(), $vars['cp_topic']->getOwnerEntity()->name, cp_translate_subtype($vars['cp_topic']->getSubtype())), 'en');
+			$cp_notify_msg_title_fr = elgg_echo('cp_notify:body_new_content:title3', array($vars['cp_topic']->getOwnerEntity()->getURL(), $vars['cp_topic']->getOwnerEntity()->name, cp_translate_subtype($vars['cp_topic']->getSubtype()), false), 'en');
+		}	$wire_post_message = "<p>{$vars['cp_topic']->description}</p>";
```
Comment (nschonni) [inline mod/cp_notifications/views/default/cp_notifications/email_template.php]: Is this supposed to be in the conditional?

Comment (pandurx) [reply mod/cp_notifications/views/default/cp_notifications/email_template.php]: yes it is!

**Comment by Steph4104** [comment]
Now I get this :
image

But I think, in french, we should read "a ajouté **un nouveau message sur le fil intitulé**"
And in englsih "posted a new wire **message entitled**"

**Comment by pandurx** [comment]
@markwooff can you look at the ajax/js calls when someone wants to reshare content to the wire? i've fixed up the notification but when i try to reshare content, the js breaks

**Comment by markwooff** [comment]
@pandurx I'm unable to replicate the ajax/js error you're receiving on https://dev.gccollab.ca - could you describe the error you're receiving?

**Comment by pandurx** [comment]
@markwooff when you attempt to reshare a content, the modal pops up, but it loads indefinitely. 
image

i made some minor changes to the thewire_image module (in the wire-message branch), but i don't know what could have affected the ajax/javascript?

**Comment by pandurx** [comment]
@markwooff the issue was because the url pointing to the javascript file was not complete

**Comment by markwooff** [comment]
@pandurx are you able to put in the fix?

**Comment by pandurx** [comment]
@markwooff yes i had to make sure all debug code and conflicts were removed and resolved

**Comment by pandurx** [comment]
@Steph4104 please test without wire images plugin enabled (as originally intended)

**Comment by Steph4104** [comment]
Here is the email I receive. 

The text in english should be: Dave posted a new wire message entitled

And in french

Dave a ajouté un nouveau message sur le fil intitulé

with the wire message without the link.

And after the wire message, I should have a view or reply link to be able to see the wire message.

To see a better exemple, you can take a look at the issue #1554 

image

**Comment by pandurx** [comment]
Part of this Pull Request was merged into Production.

**Comment by LemieuxGen** [comment]
@pandurx sorry, just catching up on github issues and saw this one that I didn't reply too. What do you need from me at this point?

**Pull request closed without merging**